tests/test_bgp_neighbor_af.rb

```diff
@@ -30,13 +30,20 @@ class TestRouterBgpNeighborAF < CiscoTestCase
   def setup
     super
     if @@reset_feat
-      config('no feature bgp', 'feature bgp')
-      config('no nv overlay evpn', 'nv overlay evpn')
+      if platform == :nexus
+        config('no feature bgp', 'feature bgp')
+        config('no nv overlay evpn', 'nv overlay evpn')
```

Comment:
Sure - I was wondering why not make the methods applicable to both platforms rather than putting Nexus configuration locally and XR configuration in a method.